src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt

```diff
@@ -69,15 +69,15 @@ open class SipMessageHandler : AbstractVerticle() {
             config.getJsonArray("exclusions")?.let {
                 exclusions = it.map(Any::toString).toSet()
             }
-            config.getJsonArray("extension-headers")?.let {
-                extensionHeaders = it.map(Any::toString).toMutableSet()
-            }
             config.getString("x-correlation-header")?.let {
                 xCorrelationHeader = it
             }
-        }
 
-        extensionHeaders.add(xCorrelationHeader)
+            extensionHeaders.add(xCorrelationHeader)
```

Comment:
If there is no `sip.message` section in config, `xCorrelationHeader` will be missed. That's why i place it outside. 
May be just remove default value for `xCorrelationHeader`?